service/k8sclient/k8client.go

```diff
@@ -295,6 +296,18 @@ func (c *K8Client) DeleteXtraDBCluster(ctx context.Context, name string) error {
 	return c.kubeCtl.Delete(ctx, res)
 }
 
+// GetXtraDBCluster get a Percona XtraDB cluster details.
+func (c *K8Client) GetXtraDBCluster(ctx context.Context, name string) (*pxc.PerconaXtraDBCluster, error) {
+	var cluster pxc.PerconaXtraDBCluster
```

Comment:
why this and Get below uses different approach?

Reply:
Removed since the check is being done in k8client